src/oop/programs_with_multiple_classes/medium/q3/Eraser.java

```diff
@@ -0,0 +1,13 @@
+package oop.programs_with_multiple_classes.medium.q3;
```

Comment:
Update this question to be consistent with the changes we discussed

Reply:
For example, this is a medium question, so you should update it so that the student codes only part of each class. Also, you can remove the toString method from both classes.